styles/_navbar.scss

```diff
@@ -1,6 +1,6 @@
 .navbar {
-  position: fixed;
   width: 100vw;
+  margin-top: 0;
```

Comment:
Is it really necessary?

Reply:
I forgot to remove that line. Justo el navbar me aparecia separado del limit de la pagina. Lo puse para ver si lo arreglaba.